### ISSUE TITLE
Cousins of nonmarried couples (half cousins)

### ISSUE THREAD

**Issue by HRN65** [vytux-com/vytux_cousins#6]
It would be nice if the module could also display cousins of nonmarried couples

**Comment by vytux-com** [comment]
Can you explain what you mean by this?

I did not think that the code cared if the parents were married or in a partnership or any other family unit, but maybe I am wrong.

Maybe you could provide an GEDCOM extract of the said family structure?

**Comment by HRN65** [comment]
If one of my grandparents had an illigitimate child, this child's children would be my cousins.
In current module they are not shown as cousins. Looks like children of half siblings are not counted.

**Comment by vytux-com** [comment]
I suppose it is technically possible to do this.

I have added some dummy data to one of me dev trees to check and it seems even webtrees does not recognise this as a cousin type of relationship.
![image](https://user-images.githubusercontent.com/5370583/70396092-c6ec2e00-1a59-11ea-9efc-ce2c8ac19fc3.png)

I will see if this is something easy to do without affecting the rest of the existing functionality

**Comment by HRN65** [comment]
Yes, you are right. In sidebar and family tab, half siblings are correctly labeled. In the Relationship chart they are not recognised as half siblings
Thank you for looking into this

**Comment by arbor95** [comment]
same to me, even closer:
my grandfather married twice.
the children of are also my uncles / aunts.
and their children should also be listed as cousins.
